clash-prelude/src/Clash/Sized/Internal/Vector.hs

```diff
@@ -0,0 +1,64 @@
+{-|
+Copyright  :  (C) 2021, QBayLogic B.V.
+License    :  BSD2 (see the file LICENSE)
+Maintainer :  QBayLogic B.V. <devops@qbaylogic.com>
+
+This module exists purely to speed up doctests of "Clash.Sized.Vector". It should
+never be imported by user code.
+-}
```

Comment:
I remember looking at this file and the other new `*.Internal` modules for `doctests` and noticing that the definitions used to be in `$setup` but could actually be moved to the doctest of a single test. I think this would make the tests for the modules quicker without needing a new module like this which I think is a _very_ bad idea.